esp-hal-common/src/rsa/mod.rs

```diff
@@ -51,6 +51,8 @@ use crate::{
 #[cfg_attr(esp32, path = "esp32.rs")]
 mod rsa_spec_impl;
 
+#[cfg(feature = "async")]
+use nb::block;
```

Comment:
Probably a bit cleaner and clearer to just remove this and invoke `nb::block!` where needed instead.